fluent-package/yum/fluent-package.spec.in

```diff
@@ -167,25 +167,51 @@ fi
 if [ -d "%{_sysconfdir}/prelink.conf.d/" ]; then
   echo "prelink detected. Installing /etc/prelink.conf.d/td-agent-ruby.conf ..."
   if [ ! -f %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf ]; then
-      %{__cp} -f /opt/td-agent/share/td-agent-ruby.conf %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf
+      %{__cp} -f /opt/fluent/share/td-agent-ruby.conf %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf
   else
     if [ $(grep '\-b /opt/td-agent/embedded/bin/ruby' -c %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf) -eq 1 ]; then
       echo "old-prelink detected, Update /etc/prelink.conf ..."
       %{__sed} -i"" %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf -e "s,/embedded/,/,"
     fi
+    if [ $(grep '\-b /opt/td-agent/bin/ruby' -c %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf) -eq 1 ]; then
+      # Migrate from td-agent v4
+      echo "old-prelink detected, Update /etc/prelink.conf ..."
+      %{__sed} -i"" %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf -e "s,/td-agent/,/fluent/,"
+    fi
   fi
 elif [ -f "%{_sysconfdir}/prelink.conf" ]; then
   if [ $(grep '\-b /opt/td-agent/embedded/bin/ruby' -c %{_sysconfdir}/prelink.conf) -eq 1 ]; then
     echo "old-prelink detected, but /etc/prelink.conf.d/ does't exist. Update /etc/prelink.conf ..."
     %{__sed} -i"" %{_sysconfdir}/prelink.conf -e "s,/embedded/,/,"
-  elif [ $(grep '\-b /opt/td-agent/bin/ruby' -c %{_sysconfdir}/prelink.conf) -eq 0 ]; then
+  elif [ $(grep '\-b /opt/fluent/bin/ruby' -c %{_sysconfdir}/prelink.conf) -eq 0 ]; then
     echo "prelink detected, but /etc/prelink.conf.d/ does't exist. Adding /etc/prelink.conf ..."
-    echo "-b /opt/td-agent/bin/ruby" >> /etc/prelink.conf
+    echo "-b /opt/fluent/bin/ruby" >> /etc/prelink.conf
+  fi
+  if [ $(grep '\-b /opt/td-agent/bin/ruby' -c %{_sysconfdir}/prelink.conf) -eq 1 ]; then
+    # Migrate from td-agent v4
+    echo "old-prelink detected, Update /etc/prelink.conf ..."
+    %{__sed} -i"" %{_sysconfdir}/prelink.conf -e "s,/td-agent/,/fluent/,"
```

Comment:
We prefer removing prelink settings completely, it's disabled by default as of RHEL 7 and not recommended.

e.g.)
* https://pagure.io/fesco/issue/1183
* https://support.oracle.com/knowledge/Oracle%20Linux%20and%20Virtualization/2658080_1.html

Reply:
It seems that this topic should be treated in another PR.

Reply:
I'll create another PR.